mackup/applications/boxer.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Boxer
+
+[configuration_files]
+Library/Preferences/net.washboardabs.boxer.plist
+Applications/DOS Games
```

Comment:
Not sure what you want to sync here.

1. Is there really an `~/Applications` folder being created?
1. Do we really want to sync whole games? It might mean a lot of big files no?

I feel like we should just sync settings.